server/app/blocks_test.go

```diff
@@ -199,11 +199,7 @@ func TestIsWithinViewsLimit(t *testing.T) {
 	t.Run("within views limit", func(t *testing.T) {
 		th.Store.EXPECT().GetLicense().Return(fakeLicense)
 
-		cloudLimit := &mmModel.ProductLimits{
-			Boards: &mmModel.BoardsLimits{
-				Views: mmModel.NewInt(2),
-			},
-		}
+		cloudLimit := &mmModel.ProductLimits{}
```

Comment:
We should remove all related to ProductLimits. The `isWithinViewsLimit` check below is all commented out and returning `true` . We can remove that as well.

Reply:
Sure.